sklearn/linear_model/least_angle.py

```diff
@@ -1338,6 +1359,15 @@ class LassoLarsIC(LassoLars):
         alphas. The alpha which has the smallest information criteria
         is chosen.
 
+    ``best_score_`` : float
+        Best cross validation score on the left out data
+        (i.e. best mean squared error). Note that by default GridSearchCV
+        computes ``best_score_`` based on the R^2, not the mean squared error,
+        which will account for differences between the two.
```

Comment:
I don't like the last part of the sentence. The difference between which two? GridSearchCV with a different score? I would just remove the "which will accout" part.